Gems/PhysX/Code/Tests/EditorTestUtilities.cpp

```diff
@@ -62,6 +63,10 @@ namespace PhysXEditorTests
         {
             editorEntity->CreateComponent<PhysX::EditorRigidBodyComponent>();
         }
+        else
```

Comment:
Probably out of scope for this change, but looks like we can probably refactor some duplicated code here.

Reply:
Yeah, let's leave that for another pass.